CONTRIBUTING.md

```diff
@@ -106,7 +106,7 @@ The most important thing to remember is to make your intention explicit. Try to
 
 ## Pull Requests
 
-New pull requests should be done either against `main` or against the related feature branch (see [Git Conventions](#branches)). You can reference the Jira task ID in the Pull Request description.
+New pull requests should be done either against `develop` or against the related feature branch (see [Git Conventions](#branches)). You can reference the Jira task ID in the Pull Request description.
 
 If your pull request is against `main` don't forget to add it to the relevant milestone. If you are not sure which one to select, use the one for the next release.
```

Comment:
`develop`?

Reply:
Yes, our default branch at the moment is `develop` this is a generic guideline to contributing to the codebase, not specific to contributing to v2 :)